meeting/meeting_november_10_2020.md

```diff
@@ -0,0 +1,158 @@
+
+**#Status_Update** - In this mode I (or someone else) will be bringing everyone up to speed on a given topic. Please save your questions/interruptions for tagged discussions or the end of the section.
+Reasonable interruptions (such as clarifying questions) are of course always welcome, but the purpose of this tag is simply to make clear that the intention of the tagged section of the agenda is to remain brief and generally speaking discussion branching from these sections should be had online, outside of the meeting (unless we all feel that it is a must-have discussion, again this is a suggestion not a commandment).
+
+**#Status_Update_Interrupt** - In this mode I (or someone else) will be bringing everyone up to speed on a given topic, but in a setting where interruptions such as questions and discussions are expected and welcome.
+Discussions here should try to remain brief.
+
+**#Discussion** - This tag explicitly opens the floor up for anyone to ask questions or make comments.
+Note that discussion tags are always optional and often times no one will have anything to say, this tag merely marks places were there is the potential for fruitfull in-meeting discussions.
+
+**#Query([names])** - This tag explicitly opens the floor up for a specific person or group of people, generally to answer a question.
+
+**#Goal([statement])** - This tag marks the agenda's intention to have a discussion with a specific purpose to achieve. These discussions may take longer.
+
+## Individual Updates (Sync)
+
+As per discussion after our previous meeting, we shall open our meeting with brief descriptions of our recent and/or current and/or expected relevant progress.
+
+**#Status_Update_Interrupt**
+
+* **#Query(Nadav)**
+* **#Query(Tibo)**
+* **#Query(Nicolas)**
+* **#Query(Ben)**
+* **#Query(Lloyd)**
+* **#Query(Antoine)**
+* **#Query(Chris)**
+* **#Query(Matt)**
+* **#Query(all)**
+
+## Secp256k1 Progress
+
+**#Status_Update**
+
+* We are still waiting on schnorrsig upstream changes to be merged into secp256k1-zkp in this [PR](https://github.com/ElementsProject/secp256k1-zkp/pull/103)
+  * There seems to have been almost no movement on this this month
+
+## Oracle Specifications
+
+**#Status_Update_Interrupt**
+
+* The initial [Oracle Proposal](https://github.com/discreetlogcontracts/dlcspecs/pull/55/) has been merged!
+  * Exact serialization and signing algorithm not fully specified by the current doc
+    * **#Discussion**
+    * https://github.com/discreetlogcontracts/dlcspecs/issues/111
+  * We need a doc for client-side oracle validation
+    * https://github.com/discreetlogcontracts/dlcspecs/issues/97
+    * Antoine has volunteered to do this
+* Any other oracle things worth discussion/mention?
```

Comment:
Tagged hashes?

Reply:
This falls under the `Exact serialization and signing algorithm` discussion